lib/reek/smells/uncommunicative_parameter_name.rb

```diff
@@ -60,10 +56,9 @@ def examine_context(ctx)
 
       private
 
-      def bad_name?(ctx, name)
-        sanitized_name = sanitize name
-        return false if sanitized_name == '*' || accept_names(ctx).include?(sanitized_name)
-        reject_patterns(ctx).any? { |pattern| sanitized_name.match pattern }
+      def acceptable_name?(context: raise, name: raise)
```

Comment:
Yes, I thought about that as well since all those "UncommunicativeFoo"-smell detectors are structurally and functionally very similar. Let's do that in a follow-up PR.